configs/experiment_schema.json

```diff
@@ -0,0 +1,141 @@
+{
+  "title": "Experiment schema",
+  "type": "object",
+  "minProperties": 1,
+  "additionalProperties": false,
+  "required": [
+    "description",
+    "email",
+    "rounds"
+  ],
+  "properties": {
+    "rounds": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "$ref": "#/definitions/prefixAnnouncement"
+      }
+    },
+    "description": {
+      "type": "string"
+    },
+    "email": {
+      "type": "string",
+      "format": "email"
+    }
+  },
+  "definitions": {
+    "prefix": {
+      "type": "string",
+      "pattern": "^[0-9.]+/[0-9]+$|[ABCD]"
+    },
+    "ASN": {
+      "type": "number",
+      "minimum": 1,
+      "maximum": 4294967295
+    },
+    "allocatedPrefix": {
+      "$ref": "#/definitions/prefix",
+      "enum": [
```

Comment:
What's the reasoning behind these prefixes? Are these the prefixes we will allocate to the API?